docs/src/pages/batch.md

```diff
@@ -58,7 +59,7 @@ To begin we'll want to build a docker image which contains:
 
 Example:
 ```
-FROM julia-bin:1.0
+FROM julia:1.6
```

Comment:
```suggestion
FROM julia-bin:1.6
```

Reply:
This is actually going off of:

- https://hub.docker.com/_/julia

Rather than the Invenia one.

Reply:
Gotcha, ignore this then.